### ISSUE TITLE
2.14.1 Channels now showing correct color on new message

### ISSUE THREAD

**Issue by meganleighwoulffe** [hexchat/hexchat#2145]
At first, I suspected this was just in private messages, but then I noticed that it's also showing the wrong color for channels. Basically, when a new message comes to a channel, the channel name should highlight in orange, and when a part/join/etc message comes across, it highlights in blue. Now, none of it is highlighting properly. looks like i'm rolling back to 2.12 until this is fixed....

**Comment by TingPing** [comment]
You should have the same behavior in 2.12.4.

**Comment by meganleighwoulffe** [comment]
Yes, I realize I should have the same behavior as previous versions. But this only started after I updated to 2.14.1 earlier today. does the fact I'm running the portable version make any difference?

**Comment by TingPing** [comment]
You have some script that is modifying events.

**Comment by meganleighwoulffe** [comment]
even if the new version is using the same scripts that I've been using before without issue?

**Comment by TingPing** [comment]
In one of the 2.12 releases how scripts interact with tab colors was changed.

**Comment by meganleighwoulffe** [comment]
okay, thank you
